msu/systems/mod_settings/elements/keybind_setting.nut

```diff
@@ -1,10 +1,4 @@
 ::MSU.Class.KeybindSetting <- class extends ::MSU.Class.StringSetting
 {
 	static Type = "Keybind";
-	// Maybe add a keybind combination validator at some point
```

Comment:
I'd keep this line in just because it's a reminder for the future

Reply:
This has been added since, see #177